index.src.html

```diff
@@ -610,87 +610,190 @@ <h2 id="mitigations">
+    Drop the feature
+  </h3>
 
-    *   <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1313580%20">Mozilla<a> and <a href="https://bugs.webkit.org/show_bug.cgi?id=164213">WebKit</a> dropped Battery Status API
-    *   <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1359076">Mozilla dropped<a> devicelight, deviceproximity and userproximity events
+  The simplest way to mitigate potential negative security or privacy impacts
+  of a feature is to drop the feature.
+  Every feature in a spec should be considered guilty of harming security
+  and/or privacy until proven otherwise.
+  Discussing dropping the feature as a mitigation for security or privacy
+  impacts is a helpful exercise as it helps illuminate the tradeoffs between
+  the feature, whether it is exposing the minimum amount of data necessary,
+  and other possibly mitigations.
 
-      <h3 id="sanitize-data">
-        Sanitize the data handled in the feature
-      </h3>
+  Every specification should seek to be as small as possible, even if only
+  for the reasons of reducing and minimizing security/privacy attack surface(s).
 
-      It is always a good strategy to consider the kinds of data a new feature is processing. For example, new features allowing the readout of data may want to adopt specific privacy strategies such as minimizing the quality of datas (quantization) or reducing the frequency, in line with standard privacy engineering practices. Examples
-      *   [BATTERY-STATUS-API] <em>“The user agent should not expose high precision readouts”</em>
-      * [SENSORS-API] <em>“Limit maximum sampling frequency”, “Reduce accuracy”</em>
+  By doing so we can reduce the overall security and privacy attack surface
+  of not only a particular feature, but of a module (related set of
+  features), a specification, and the overall web platform.
 
+  Examples
 
-      <h3 id="user-mediation">
-        Making a privacy impact assessment
-      </h3>
+  *   <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1313580%20">Mozilla<a> and <a href="https://bugs.webkit.org/show_bug.cgi?id=164213">WebKit</a> dropped Battery Status API
+  *   <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1359076">Mozilla dropped<a> devicelight, deviceproximity and userproximity events
 
-      Some features are potentially supplying very sensitive data, and it is the end-developer,
-      system owners, or managers responsibility to realize this and act accordingly in the design of his/her
-      system. Some use may warrant conducting as privacy impact assessment, especially when data relating to
-      individuals may be processed. Examples.
+  <h3 id="privacy-impact-assessment">
+    Making a privacy impact assessment
+  </h3>
 
-      *   [GENERIC-SENSORS] advices to consider performing of a privacy impact assessment
+  Some features are potentially supplying very sensitive data, and it is
+  the end-developer, system owners, or managers responsibility to realize
```

Comment:
Suggest rewording to "..the responsibility of the end-developer, system owner, or manager to realize..." to avoid making them all possessive

Reply:
Thanks! This is addressed in 444a3f0.